spec/models/document_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "spec_helper"
+
+describe Document do
+  describe "initialization" do
+    let(:attrs) do
+      {
+        title: 'A title',
+        link: 'link.com'
+      }
+    end
+    let(:finder) do
+      double('finder', date_metadata_keys: [], text_metadata_keys: [])
+    end
+    let(:document) { described_class.new(attrs, finder) }
```

Comment:
@tijmenb I was following the same naming as in the main class so we could follow things along. I will rename in both places.